modules/doc/content/newsletter/2025/2025_11.md

```diff
@@ -1,22 +1,87 @@
+[ArrayReactionNodalKernel.md]. Hand-coded Jacobian array nodal kernel support was introduced
+in [2025_10.md].
+
+### Mesh generation
+
+The [PolyLineMeshFollowingNodeSetGenerator.md] has been added to generate a 1D line mesh, of `edge` elements, by averaging
+the positions of nodes in a multi-dimensional nodeset. Several options and heuristics have been implemented to further define the averaging process.
+
+The [XYZDelaunayGenerator.md] now has an option to only convert the boundary of a `hole` mesh to tetrahedrals, rather than the entire `hole` mesh.
+A `hole` mesh is a mesh that is pre-existing and forming a hole inside the Delaunay tetrahedralization (and thus does not need to be tetrahedralized).
+
+### KokkosFunction
+
+The [KokkosFunction](syntax/KokkosFunctions/index.md) system was added to deliver the same capability as `Functions` but in Kokkos-based
+finite element simulations. As a reminder, `Functions` are intended to be known functions of time and space exclusively, and not depend
+on material properties or variable values.
+
 ### DiscreteVariableResidualNorm now uses the un-scaled residual norm
 
 [DiscreteVariableResidualNorm.md] has added the parameter [!param](/Postprocessors/DiscreteVariableResidualNorm/include_scaling_factor), which allows the user to control whether the residual norm includes the scaling factor or not. Before, the scaling factor was always included, but now the default behavior is to not include the scaling factor. This is ideal for the most common use case of this object, which is convergence monitoring, which usually should be independent of the scaling factor.
 
 ## MOOSE Modules Changes
 
+### Navier Stokes module
+
+The Physics syntax for the weakly compressible Navier Stokes equations can now accept a `fp` parameter for the fluid properties
+being used. This helps define the functor materials needed to compute fluid properties in the kernels forming the equations.
+
+### Porous flow module
+
+Two new `ActionComponents` were created to add injection and production wells in a compact syntax.
+These wells are represented by multiple objects to represent the flow channel from the well, and the boundary condition
+at the well.
+
+### Heat transfer module
+
+The P1 radiation model was added with the linear finite volume discretization. The use of this discretization facilitates
+coupling convective flow problems, solved notably in the Navier Stokes module using this discretization, directly alongside
+radiative heat transfer.
+
+### Subchannel module
+
+A new [SCMClosure system](syntax/SCMClosures/index.md) was added to allow for user-developers to specify the closures to use
+in the subchannel solve. For now, only the friction factors can be selected in this system. Developers may create new closures
+in this system, with a dedicated base class for each physical quantity computed using a closure.
```

Comment:
I think this could be made more clear. You're saying that I'd need two different base classes for computing friction factors vs. heat transfer coefficients?